static/bidder-info/globalsun.yaml

```diff
@@ -0,0 +1,18 @@
+endpoint: "http://endpoint.globalsun.io/pserver"
+maintainer:
+  email: "prebid@globalsun.io"
+capabilities:
+  site:
+    mediaTypes:
+    - banner
+    - video
+    - native
+  app:
+    mediaTypes:
+    - banner
+    - video
+    - native
+userSync:
+  redirect:
+    url: "https://cs.globalsun.io/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redir={{.RedirectURL}}"
+    userMacro: "[UID]"
```

Comment:
I get an error "No GDPR/CCPA consent" and http 400 status when testing usersync. Here is my test url: https://cs.globalsun.io/pbserver?gdpr=&gdpr_consent=CPfCRQAPfCRQAAAAAAENCgCAAOAAAAAAAAAAAQAAAAAEAIAAAAAAAAAA&us_privacy=1NYN&redir=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dglobalsun%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D

Reply:
400 status code with the text "No GDPR/CCPA consent" is returned if we receive invalid GDPR_content in a request from the European area. In that case, we don't sync